recipe/meta.yaml

```diff
@@ -13,18 +13,25 @@ build:
 requirements:
   build:
     - python
+    - setuptools
     - numpy
     - matplotlib
 
   run:
     - python
+    - setuptools
     - numpy
     - matplotlib
     - nose
 
 test:
   imports:
     - probscale
+    
+  requires:
+    - nose
+    - matplotlib
```

Comment:
If matplotlib was a runtime dependency anyway, this is redundant. Looks like you do need to make scipy a test dependency though.